usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -108,10 +109,15 @@ def gather_shared_award_data(self, data, award_type):
                 continue
 
             # Get or create unique DUNS-recipient pair
-            recipient, created = LegalEntity.objects.get_or_create(
-                recipient_unique_id=row['duns'],
-                recipient_name=recipient_name
-            )
+            try:
+                recipient, created = LegalEntity.objects.get_or_create(
+                    recipient_unique_id=row['duns'],
+                    recipient_name=recipient_name
+                )
+            except MultipleObjectsReturned:
+                created = False
+                print('Legal Entity with DUNS: {} and name: {} returned two rows. '
+                      'Skipping...'.format(row['duns'], recipient_name))
```

Comment:
This isn't actually skipping anything, you need to put a `continue` after this to skip this record. If you want to get the first one you have to do something else. If you want a NULL in this case you have to set recipient to NULL